cpu/nrf52/Makefile.features

```diff
@@ -25,4 +25,11 @@ FEATURES_PROVIDED += cortexm_mpu
 
 FEATURES_PROVIDED += periph_i2c_reconfigure
 
+# On top of the default 1Mbit PHY mode, all nrf52 support the 2MBit PHY mode,
+# and the 52840 does further support the coded PHYs
+FEATURES_PROVIDED += ble_phy_2mbit
+ifneq (,$(filter nrf52840% nrf52811%,$(CPU_MODEL)))
```

Comment:
nrf52811 supports it but nrf52810 does not?

Reply:
So far I copied this statement from (an apparently older version of) `boards/common/nrf52/Makefile.dep`.

Will re-check and verify which of the family members actually support the coded PHY...

Reply:
> nrf52811 supports it but nrf52810 does not?

finally checked, and yes, that is the case: https://infocenter.nordicsemi.com/topic/struct_nrf52/struct/nrf52.html